codex-rs/mcp-server/src/codex_tool_runner.rs

```diff
@@ -131,7 +131,8 @@ pub async fn run_codex_tool_session(
                             });
                         }
 
-                        break;
+                        // Continue, don't break so the session continues.
```

Comment:
Should we keep this comment?

Reply:
I think the comments are somewhat useful because some branches continue and some break and when the next person implements a branch, they could make the same mistake as you did the first time since it's not totally obvious